src/Sylius/Component/Attribute/Model/AttributeValue.php

```diff
@@ -87,47 +87,47 @@ public function getId()
     /**
      * {@inheritdoc}
      */
-    public function getSubject()
+    public function getSubject(): ?AttributeSubjectInterface
     {
         return $this->subject;
     }
 
     /**
      * {@inheritdoc}
      */
-    public function setSubject(AttributeSubjectInterface $subject = null)
+    public function setSubject(?AttributeSubjectInterface $subject = null): void
```

Comment:
Calling `setSubject()` without arguments makes no sense, how about removing the default value and adding an entry in upgrade.md?